src/main/target/common_fc_pre.h

```diff
@@ -128,3 +128,5 @@
 #if (FLASH_SIZE > 256)
 #define USE_UNCOMMON_MIXERS
 #endif
+
+#define USE_RCSPLIT
```

Comment:
Wow, you are so cautious! I'll recommit with this until  #2856 finished. Thank you .